nearquake/test/test_utils.py

```
@@ -0,0 +1,383 @@
+
+def test_generate_date_range():
+    assert generate_date_range(
+        start_date="2023-01-01", end_date="2023-02-01", interval=31
+    ) == [(datetime.datetime(2023, 1, 1, 0, 0), datetime.datetime(2023, 2, 1, 0, 0))]
+
+
+def test_generate_date_range_negative():
+
+    generate_date_range(
+        start_date="2023-01-01", end_date="2022-01-01", interval=15
+    ) == []
```

Ruff (0.8.2)

Pointless comparison at end of function scope. Did you mean to return the expression result?

(B015)